Documentation/Modules.rst

```diff
@@ -0,0 +1,10 @@
+Modules
```

Comment:
This should also go under the Developer's Guide.  Users won't care about this.

Reply:
merged in a21efb7